examples/quantization_w8a8_fp8/README.md

```diff
@@ -9,9 +9,7 @@
 To get started, install:
 
 ```bash
-git clone https://github.com/vllm-project/llm-compressor.git
-cd llm-compressor
-pip install -e .
+pip install llmcompressor
```

Comment:
I think we should pin the version so it is clear when this was made/updated

Reply:
good idea